contracts/tokens/CurveTokenV1.vy

```diff
@@ -0,0 +1,171 @@
+# @version 0.1.0b16
```

Comment:
Prefer the use of [`brownie-token-tester`](https://github.com/iamdefinitelyahuman/brownie-token-tester) over including token contracts for testing.  It keeps the repo a bit cleaner.